fla/layers/mom_varlen.py

```
@@ -0,0 +1,724 @@
+                "Expected attention_mask as a 0-1 matrix with shape [batch_size, seq_len] "
+                "for padding purposes (0 indicating padding). "
+                "Arbitrary attention masks of shape [batch_size, seq_len, seq_len] are not allowed."
+            )
+
+        mode = 'fused_recurrent' if hidden_states.shape[1] <= 64 else self.mode
+        if self.training:
+            assert mode == 'chunk', "Only chunk mode is supported in training."
+
+        if self.use_short_conv:
+            conv_mask = attention_mask[:, -hidden_states.shape[1]:] if attention_mask is not None else None
+            seq_idx=kwargs.get('seq_idx', None)
```

Ruff (0.11.9)

Use `kwargs.get('seq_idx')` instead of `kwargs.get('seq_idx', None)`

Replace `kwargs.get('seq_idx', None)` with `kwargs.get('seq_idx')`

(SIM910)